ext/nnstreamer/tensor_decoder/box_properties/mobilenetssdpp.cc

```diff
@@ -189,6 +190,7 @@ MobilenetSSDPP::checkCompatible (const GstTensorsConfig *config)
 {
   const uint32_t *dim1, *dim2, *dim3, *dim4;
   int locations_idx, classes_idx, scores_idx, num_idx, i;
+  guint max_detection;
```

Comment:
How about changing the code to store the dim[2] value directly in this->max_detection?
And remove guint max_detection.

Reply:
Removed `max_detection` and use dim value directly!
Thank you :)